crates/ironrdp-server/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+#[macro_use]
+extern crate tracing;
+
+mod builder;
+mod capabilities;
+mod display;
+mod encoder;
+mod handler;
+mod server;
+
+pub use display::*;
+pub use handler::*;
+pub use server::*;
+
+pub use ironrdp_acceptor as acceptor;
```

Comment:
Should we re-export `ironrdp_acceptor` from `ironrdp_server` or from `ironrdp`? I feel `ironrdp` should re-export it.